.github/workflows/job-test-in-host.yml

```
@@ -105,7 +108,18 @@ jobs:
             echo "::group:: configure cdi for docker"
             sudo mkdir -p /etc/docker
             sudo jq '.features.cdi = true' /etc/docker/daemon.json | sudo tee /etc/docker/daemon.json.tmp && sudo mv /etc/docker/daemon.json.tmp /etc/docker/daemon.json
-            sudo systemctl restart docker
+            echo "::endgroup::"
+            echo "::group:: downgrade docker to the specific version we want to test (${{ inputs.docker-version }})"
+            sudo apt-get update -qq
+            sudo apt-get install -qq ca-certificates curl
+            sudo install -m 0755 -d /etc/apt/keyrings
+            sudo cp ./hack/provisioning/docker/gpg /etc/apt/keyrings/docker.asc
```

```suggestion
            sudo cp ./hack/provisioning/gpg/docker /etc/apt/keyrings/docker.asc
```

So as to collect all the gpg keys in the same directory